src/utils/list.js

```diff
@@ -1,4 +1,13 @@
-export const range = (count, start = 0) => count <= 0 ? [] : Array(count).fill().map((_, i) => i + start);
+export const range = (count, start = 0) => {
```

Comment:
Side note, but isn't range usually `range(start, end[, step])`?

Reply:
Yeah, I didn't include a step because it isn't necessary for these DASH segment indexes. I'll rename the function if there's a more precise name.

Reply:
Looks good as is, though I think `start` and `end` is usually the standard